Utils.py

```diff
@@ -57,7 +57,7 @@ def Write_Performance(output_path=output_path, NN=NN):
             print("Early stopping = " + str(early_stopping),file=perf)
 
 ### calculate the yearly relevant comment counts
-def Get_Counts(path=path, random=False, frequency="monthly"):
+def Get_Counts(model_path=model_path, random=False, frequency="monthly"):
```

Comment:
I believe Get_Counts was moved to within ModelEstimation.py in the commits above. Utils.py should be adjusted accordingly.

Reply:
This was resolved in our latest joint debugging session. It can wait until model_improvements is merged with Dev I guess

Reply:
Yes